### DISCUSSION TITLE
Display compatible with ableton.

### DISCUSSION THREAD

**Discussion by Trickster-git** [tttapa/Control-Surface#931] [Ideas]
Hi, Pieter!
It`s possible add ability to print on display Ableton track name, device parameter name and value, like on push display.
like that idea but in hardwire
https://github.com/zsteinkamp/m4l-zs-Knobbler3

**Comment by tttapa** [comment]
It'll depend on whether Ableton somehow sends the data you're interested in. You can use a MIDI monitor tool to find out.

The link you posted seems to use OSC, not MIDI, so it may not be possible.
